source/dofs/dof_handler_policy.cc

```diff
@@ -162,21 +163,24 @@ namespace internal
                   case 0:
                     {
                       identities = std::make_unique<DoFIdentities>(
-                        fe1.hp_vertex_dof_identities(fe2));
+                        fes[fe_index_1].hp_vertex_dof_identities(
+                          fes[fe_index_2]));
```

Comment:
You could've also introduced element references above to avoid touching historic code :-)
```
const auto &fe1 = fes[fe_index_1];
const auto &fe1 = fes[fe_index_2];
```

Reply:
I have plans to get rid of this code, so didn't bother.